beacon-chain/db/deposits.go

```diff
@@ -82,6 +83,27 @@ func (db *BeaconDB) AllDeposits(ctx context.Context, beforeBlk *big.Int) []*pb.D
 	return deposits
 }
 
+// DepositsContainersTillBlock returns the deposit containers sorted by block height till block height.
```

Comment:
what do you mean by `sorted block height till block height`? till highest block height?

Reply:
changed to 
// DepositsContainersTillBlock returns the deposit containers sorted by block height where
// block height lower then beforeBlk.